src/modules/72-templates-library/pages/TemplatesPage/views/TemplatesListView/TemplatesListView.tsx

```diff
@@ -189,16 +180,13 @@ export const TemplatesListView: React.FC<TemplatesViewProps> = (props): JSX.Elem
         width: '5%',
         Cell: RenderColumnMenu,
         disableSortBy: true,
-        onPreview,
-        onOpenEdit,
-        onOpenSettings,
-        onDelete
+        onPreview
       }
     ],
-    [isGitSyncEnabled, onPreview, onOpenEdit, onOpenSettings, onDelete]
+    [isGitSyncEnabled, onPreview]
```

Comment:
Codacy has a fix for the issue: [React Hook React.useMemo has missing dependencies: 'getString' and 'getTemplateNameWidth'. Either include them or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9273329)
```suggestion
    [getString, getTemplateNameWidth, isGitSyncEnabled, onPreview]
```